_tests/navigation/index.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: Navigation
+parent: Tests Home
+---
+
+# Navigation
```

Comment:
A bit concerned about how brittle the links are to things moving around, but I have no strong opinions.

Reply:
I don't see why you're concerned. If the tests are moved to a separate repo without changing the file titles, the links should automatically remain valid, since the navigation hierarchy for a Just the Docs website is independent of where the files are located in its repo.